server/auth.go

```diff
@@ -264,7 +297,48 @@ func (s *Server) isClientAuthorized(c *client) bool {
 	}
 	s.mu.Unlock()
 
-	// Verify the signature against the nonce.
+	// If we have a jwt and a userClaim, make sure we have the Account, etc associated.
+	// We need to look up the account. This will use an account resolver if one is present.
+	if juc != nil {
+		if acc = s.LookupAccount(juc.Issuer); acc == nil {
+			c.Debugf("Account JWT can not be found")
+			return false
+		}
+		if !s.isTrustedIssuer(acc.Issuer) {
+			c.Debugf("Account JWT not signed by trusted operator")
+			return false
+		}
+		if acc.IsExpired() {
```

Comment:
I don't think we will make it available if it fails the jwt.IsBlocking(true) since we process it that way when first handed the claims IIRC. The acc.IsExpired is for us when we fire the timer to mark the account. Also triggers us to refetch at some point.